README.md

```diff
@@ -29,6 +29,28 @@ After this there should be a `nextcloud-exporter` binary in your current directo
 
 ## Client credentials
 
+The exporter supports two different approaches for authenticating with the Nextcloud server:
+
+- Token authentication (needs Nextcloud 22 or newer)
+- Username and password
+
+If you have Nextcloud 22 then using the token authentication is recommended, because it does not need a normal user account with admin privileges.
+
+### Token authentication
+
+Nextcloud 22 and newer versions support "token authentication" for the serverinfo. That way, accessing this information does not need a normal user account with admin privileges. You can set the token to anything you like, but the recommendation is to set it to a long random number:
+
+```bash
+# Generate random value (for example using openssl)
+TOKEN=$(openssl rand -hex 32)
+# Set token (using the occ console application)
+occ config:app:set serverinfo token --value "$TOKEN"
+```
+
+You can then use this generated token in the exported configuration instead of username and password.
```

Comment:
Maybe add a line like: `The auth token takes preference over username and password authentication`

Reply:
I have added a comment about this in the previous section now.